### PR TITLE
Add basic info about new machine learning services

### PR DESCRIPTION
**Pull request by artemnikitin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -304,6 +307,9 @@ Many services within AWS can at least be compared with Google Cloud offerings or
...
 | Managing SSL/TLS certificates            | Certificate Manager                                                                  |                |                 |                 | Let's Encrypt, Comodo, Symantec, GlobalSign |
+| Automatic speech recognition and natural language understanding            | Lex   | Natural Language API               |                 | Cognitive services                | AYLIEN Text Analysis API, Ambiverse Natural Language Understanding API  |Stanford's Core NLP Suite, Apache OpenNLP, Apache UIMA |
+| Text-to-speech engine in the cloud            | Polly                                                                  |                |                 |                 |Nuance, Vocalware, IBM Watson |Mimic, eSpeak, MaryTTS |
+| Image recognition            | Rekognition                                                            |   Vision API              |                |Cognitive services                 | IBM Watson, Clarifai |TensorFlow, OpenCV |
```
Comment (lukedeo) [inline README.md]: Would recommend adding both Microsoft and Google Cloud's Vision APIs.

File: README.md [hunk 2]
```diff
@@ -304,6 +307,9 @@ Many services within AWS can at least be compared with Google Cloud offerings or
 | Mobile app analytics          | Mobile Analytics                                                             | Firebase Analytics           |                 | HockeyApp                          | Mixpanel                          |                                                            |
 | Mobile app testing            | Device Farm                                                                  | Firebase Test Lab            |                 | Xamarin Test Cloud                 | BrowserStack, Sauce Labs, Testdroid                                                            |
 | Managing SSL/TLS certificates            | Certificate Manager                                                                  |                |                 |                 | Let's Encrypt, Comodo, Symantec, GlobalSign |
+| Automatic speech recognition and natural language understanding            | Lex   | Natural Language API               |                 | Cognitive services                | AYLIEN Text Analysis API, Ambiverse Natural Language Understanding API  |Stanford's Core NLP Suite, Apache OpenNLP, Apache UIMA |
```
Comment (lukedeo) [inline README.md]: Would recommend adding Google Cloud's Speech/NLU APIs. Also consider [spaCy](http://spacy.io) for the "build your own".

File: README.md (hunk 1 above)
Comment (artemnikitin) [inline README.md]: I added them, Vision API and Cognitive services respectively.

File: README.md (hunk 2 above)
Comment (artemnikitin) [inline README.md]: I added it, look Natural Language API. I will add spaCy shortly.

Comment (artemnikitin) [inline README.md]: spaCy was added

**Comment by jlevy** [comment]
👍 Great. Thanks @artemnikitin @lukedeo !

**Pull request merged**